source/agora/consensus/EnrollmentManager.d

```diff
@@ -965,36 +965,39 @@ unittest
     auto utxo_hash1 = utxo_hashes[0];
     auto enrollment = man.createEnrollment(utxo_hash1);
     assert(man.pool.add(enrollment, &storage.findUTXO));
-    assert(man.getValidatorCount(block_height) + 1 == 1);
+    assert(man.getValidatorCount(block_height) == 0);  // not active yet
```

Comment:
Hum, if it's using  the test genesis  block, is that correct ?

Reply:
There's no ledger here that loads the enrollment manager with the validator set.